metashare/api/views.py

```diff
@@ -171,7 +171,14 @@ def review(self, request, pk=None):
                 serializer.errors, status=status.HTTP_422_UNPROCESSABLE_ENTITY
             )
         task = self.get_object()
-        task.queue_submit_review(user=request.user, data=serializer.data)
+        org = serializer.validated_data["org"]
+
+        if not task.pr_is_open:
+            raise ValidationError(_("The pull request for this task has been closed."))
+        if not (org or task.review_valid):
+            raise ValidationError(_("Cannot submit review without a Review Org."))
```

Comment:
These will result in `500` errors if they are raised here.

Reply:
Oh? We're doing it similarly above in `CreatePrMixin` and `ScratchOrgViewSet`, and in testing it seems to result in a `400`.

Reply:
Oh, hm. I wonder if DRF handles `ValidationError` even in a view specially. Well, OK!

Reply:
Yeah, seems like it: https://www.django-rest-framework.org/api-guide/exceptions/

> The handled exceptions are:
> - Subclasses of `APIException` raised inside REST framework.
> - Django's `Http404` exception.
> - Django's `PermissionDenied` exception.